artemis-cli/src/test/java/org/apache/activemq/artemis/cli/commands/CreateTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.cli.commands;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+
+import org.apache.activemq.artemis.utils.XmlProvider;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+@RunWith(Parameterized.class)
+public class CreateTest {
+
+   private final String testName;
+   private String httpHost;
+   private boolean relaxJolokia;
+
+   public CreateTest(String testName, String httpHost, boolean relaxJolokia) {
+      this.testName = testName;
+      this.httpHost = httpHost;
+      this.relaxJolokia = relaxJolokia;
+   }
+
+   @Parameters
```

Comment:
```suggestion
   @Parameterized.Parameters(name = "{0}")
```
The test name could be used to improve test description